daemon/src/taker_cfd.rs

```diff
@@ -67,22 +67,22 @@ enum RollOverState {
     None,
 }
 
-pub struct Actor {
+pub struct Actor<O, M> {
     db: sqlx::SqlitePool,
     wallet: Wallet,
     oracle_pk: schnorrsig::PublicKey,
     cfd_feed_actor_inbox: watch::Sender<Vec<Cfd>>,
     order_feed_actor_inbox: watch::Sender<Option<Order>>,
     update_cfd_feed_sender: watch::Sender<UpdateCfdProposals>,
-    send_to_maker: Address<send_to_socket::Actor<wire::TakerToMaker>>,
-    monitor_actor: Address<monitor::Actor>,
+    send_to_maker: Box<dyn MessageChannel<wire::TakerToMaker> + Send>,
```

Comment:
I thought `MessageChannel` already enforces `Send`?

Reply:
Good catch. This was a failed attempt at solving the `MessageSink` problem and I forgot to revert it.